src/ctap/mod.rs

```diff
@@ -931,22 +931,64 @@ where
         (self.check_user_presence)(cid)?;
 
         // Sanity checks
-        let has_priv_key = self.persistent_store.attestation_private_key()?.is_some();
-        let has_cert = self.persistent_store.attestation_certificate()?.is_some();
+        let current_priv_key = self.persistent_store.attestation_private_key()?;
+        let current_cert = self.persistent_store.attestation_certificate()?;
 
-        if params.attestation_material.is_some() {
+        let response = if params.attestation_material.is_some() {
             let data = params.attestation_material.unwrap();
```

Comment:
Please change this to:
`let response = if let Some(data) = params.attestation_material {`

Reply:
Not applicable after applying the modification suggested by Julien